src/cloud_provider/digitalocean/router.rs

```diff
@@ -522,34 +522,24 @@ impl Create for Router {
         send_progress_on_long_task(
             self,
             crate::cloud_provider::service::Action::Create,
-            Box::new(|| delete_stateless_service(target, self, true)),
+            || delete_stateless_service(target, self, true),
         )
     }
 }
 
 impl Pause for Router {
-    fn on_pause(&self, target: &DeploymentTarget) -> Result<(), EngineError> {
-        info!("DO.router.on_pause() called for {}", self.name());
-
-        send_progress_on_long_task(
```

Comment:
why did you remove all send progress?

Reply:
As discussed, we don't want to delete router when we send a PAUSE, we basically it just doing nothing, a NOP action